utils.go

```diff
@@ -63,18 +63,115 @@ func IsDotFile(path string) bool {
 	return strings.HasPrefix(base, ".") && base != "."
 }
 
-var isVendorRegExp *regexp.Regexp
+// func IsVendor(filename string) bool {
```

Comment:
This commented out code should be removed

Reply:
It is handy to have it for benchmarking purpose. Moved it to commit message instead, history re-written.